integration/core/test_cli.py

```diff
@@ -751,21 +754,22 @@ def backup_core(bin, engine_manager_client,  # NOQA
     os.rename(volume_cfg_path, volume_tmp_cfg_path)
     assert os.path.exists(volume_tmp_cfg_path)
 
-    volume_info = cmd.backup_volume_list(grpc_controller_client.address,
-                                         "", backup_target)
-    assert MESSAGE_TYPE_ERROR in volume_info[VOLUME_NAME]["Messages"]
+    url = get_backup_volume_url(backup_target, VOLUME_NAME)
+    with pytest.raises(subprocess.CalledProcessError):
+        cmd.backup_inspect_volume(grpc_controller_client.address, url)
 
     os.rename(volume_tmp_cfg_path, volume_cfg_path)
     assert os.path.exists(volume_cfg_path)
 
-    volume_info = cmd.backup_volume_list(grpc_controller_client.address,
-                                         "", backup_target)
-    assert volume_info[VOLUME_NAME]["Messages"] is not None
-    assert MESSAGE_TYPE_ERROR not in volume_info[VOLUME_NAME]["Messages"]
+    url = get_backup_volume_url(backup_target, VOLUME_NAME)
+    volume_info = cmd.backup_inspect_volume(
+        grpc_controller_client.address, url)
+    assert volume_info["Messages"] is not None
+    assert MESSAGE_TYPE_ERROR not in volume_info["Messages"]
 
     # backup doesn't exists so it should error
+    url = backup_target + "?backup=backup-unk" + "&volume=" + VOLUME_NAME
```

Comment:
Any reason this was moved from the inner scope?

Reply:
can't remember, revert it.